graphx/src/main/scala/org/apache/spark/graphx/VertexRDD.scala

```diff
@@ -154,7 +154,30 @@ abstract class VertexRDD[VD](
    * @return a VertexRDD containing the results of `f`
    */
   def leftZipJoin[VD2: ClassTag, VD3: ClassTag]
-      (other: VertexRDD[VD2])(f: (VertexId, VD, Option[VD2]) => VD3): VertexRDD[VD3]
+      (other: VertexRDD[VD2])
+      (f: (VertexId, VD, Option[VD2]) => VD3)
+    : VertexRDD[VD3]
+
+  /**
+   * Left joins this RDD with another VertexRDD with the same index. This function will fail if
+   * both VertexRDDs do not share the same index. The resulting vertex set contains an entry for
```

Comment:
Very true. Rereading the docs it looks like they haven't been updated with verbage of a few of the more recent bug fixes. Will add that.